test/common/index.js

```diff
@@ -802,3 +802,32 @@ exports.hijackStdout = hijackStdWritable.bind(null, 'stdout');
 exports.hijackStderr = hijackStdWritable.bind(null, 'stderr');
 exports.restoreStdout = restoreWritable.bind(null, 'stdout');
 exports.restoreStderr = restoreWritable.bind(null, 'stderr');
+
+// Check if the executable links to the shared lib node or not.
+exports.isSharedLibNode = process.config.variables.node_shared === true;
```

Comment:
This is currently not used at all, right? If so, I would rather remove it again.

Reply:
sure!